client.js

```diff
@@ -83,6 +83,135 @@ async function sqlExample(hzClient) {
     process.exit(0);
 }
 
+async function jsonSerializationExample(hzClient) {
+    await createMappingForCountries(hzClient);
+
+    await populateCountriesMap(hzClient);
+
+    await selectAllCountries(hzClient);
+
+    await createMappingForCities(hzClient);
+
+    await populateCityMap(hzClient);
+
+    await selectCitiesByCountry(hzClient, "AU");
+
+    await selectCountriesAndCities(hzClient);
+
+    process.exit(0);
+}
+
+async function createMappingForCountries(hzClient) {
+    // see: https://docs.hazelcast.com/hazelcast/5.0/sql/mapping-to-maps#json-objects
+    console.log("Creating mapping for countries...");
+
+    const mappingQuery =
```

Comment:
Can we replace this with multiline string in order to improve readability?
```
    const mappingQuery = `
        CREATE OR REPLACE MAPPING country(
         __key VARCHAR,
         isoCode VARCHAR,
         country VARCHAR)
         TYPE IMap
         OPTIONS (
             'keyFormat' = 'varchar',
             'valueFormat' = 'json-flat'
         )
    `
```

Reply:
What js version are we aiming for?

Reply:
What I figured out from here: https://github.com/hazelcast/hazelcast-nodejs-client/blob/master/package.json
```
"engines": {
    "node": ">=10.4.0"
}
```

Reply:
But there is no NodeJS version mentioned in https://github.com/hazelcast/hazelcast-cloud-nodejs-sample-client/blob/master/package.json
Is it how it should be?

Reply:
In any case node 10.4 supports template literals:
https://node.green/#ES2015-syntax-template-literals
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
Fixed multiline strings.

Reply:
Yes, we need at least Node 10 to run the client, so the applications that are using the client should be at least as new as this. So, I would also suggest adding that constraint here

Reply:
Thanks, added.